build/stage-build.yml

```diff
@@ -121,6 +121,13 @@ jobs:
       value: net6.0-ios
     - name: ArtifactName
       value: $(iOSArtifactName)_${{ parameters.applicationEnvironment }}
+    - name: BannerVersionNameColor # determines the color of the banner to apply based on env (can only be defined outside template)
+      ${{ if eq( parameters['applicationEnvironment'], 'Staging' ) }}: 
```

Comment:
Shouldn't we have a separate color to distinct canary?

Reply:
Canary color is #FFEF00 :)